### DISCUSSION TITLE
Just the inputs and not the form

### DISCUSSION THREAD

**Discussion by ghost** [rjsf-team/react-jsonschema-form#4115] [Q&A]
Is it possible to grab just the inputs from this and not the submittable form part?

I have a use case where I want to have a section of a bigger form be generated dynamically given a JSON schema and this would work perfectly for that idea if I can use it to only generate the input fields without the surrounding form and submit button.

**Comment by heath-freenome** [comment]
Yes, you can provide an alternate React element to render in place of form using the [tagName](https://rjsf-team.github.io/react-jsonschema-form/docs/api-reference/form-props/#tagname) prop on `Form`. I would recommend implementing a React  component that handles all of the props passed to `FormTag` (see the code for it [here](https://github.com/rjsf-team/react-jsonschema-form/blob/a7b25e8a1803149eccc4fd175ab6412d17cdf77c/packages/core/src/components/Form.tsx#L964))
